cli/program_state.rs

```diff
@@ -103,8 +104,13 @@ impl ProgramState {
       None => None,
     };
 
+    let coverage_dir = flags
+      .coverage_dir
+      .clone()
+      .or_else(|| env::var("DENO_COVERAGE_DIR").ok());
```

Comment:
Why do we need both flag and env variable?

Reply:
We can make due with just an environment variable (the environment variable is needed for persisting between processes) but the general consensus in discord was that people preferred to keep `deno test --coverage <dir>` as the entry point for collecting coverage with deno test.